### DISCUSSION TITLE
Error: Jacobian mis:allocation

### DISCUSSION THREAD

**Discussion by aryakeyvan-star** [google-deepmind/mujoco#2281] [Asking for Help]
### Intro

Hi!

I am a graduate student using MuJoCo 3.2.3 on ubuntu 

### My setup

Mujoco v3,2,3

### My question

I am having the same issue as #2187 . I am getting (Error: engine error: mj_makeConstraint: constraint Jacobian mis-allocation: found nnzJ=72 but allocated 48). 

Are there any workarounds to solve this issue and when will the bug fix happen?

### Minimal model and/or code that explain my question

If you encountered the issue in a complex model, please simplify it as much as possible (while still reproducing the issue).

Model:
<details>
<summary>minimal XML</summary>

```XML

<mujoco>
  <worldbody>
    <light pos="0 0 1"/>
    <geom type="sphere" size="1" rgba="1 0 0 1"/>
  </worldbody>
</mujoco>

```
</details>

Code:
```python
import mujoco
import mediapy as media
model = mujoco.MjModel.from_xml_string(xml)
data = mujoco.MjData(model)

with mujoco.Renderer(model) as renderer:
  mujoco.mj_forward(model, data)
  renderer.update_scene(data)

  media.show_image(renderer.render())
```

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by yuvaltassa** [comment]
The solution is in the issue you referenced and [in the 3.2.5 changelog](https://mujoco.readthedocs.io/en/latest/changelog.html#id4): upgrade to version >=3.2.5
